### PR TITLE
task #2787:- youth-net UI Implementation

### PR DESCRIPTION
**Pull request by shreyas1434shinde**

### CODE REVIEW

**Comment by shreyas1434shinde** [comment]
YN-Dev-Mentors-–-Figma-12-19-2024_03_18_PM

**Comment by shreyas1434shinde** [comment]
YN-Dev-Mentors-–-Figma-12-19-2024_09_49_PM (1)

**Comment by shreyas1434shinde** [comment]
YN-Dev-Mentors-–-Figma-12-19-2024_09_49_PM

File: src/components/youthNet/Surveys.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <Typography
+                    sx={{
+                        fontSize: '14px',
+                        fontWeight: 500,
+                        lineHeight: '20px',
+                        color: '#7C766F',
+                    }}
+                    className="one-line-text"
+                >
+                    Closed on {date}
```
Comment (itsvick) [inline src/components/youthNet/Surveys.tsx:41]: @shreyas1434shinde translation

**Pull request merged**